metrics_utils/src/lib.rs

```diff
@@ -230,6 +242,319 @@ impl BackfillerMetricsConfig {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct Peer2PeerConsistencyMetricsConfig {
+    pub db_bubblegum_get_grand_epochs_latency: Histogram,
+    pub db_bubblegum_get_epochs_latency: Histogram,
+    pub db_bubblegum_get_changes_latency: Histogram,
+    pub db_account_get_grand_buckets_latency: Histogram,
+    pub db_account_get_buckets_latency: Histogram,
+    pub db_account_get_latests_latency: Histogram,
+
+    pub bbgm_consistency_grpc_client: Arc<BubblegumConsistencyGrpcClientMetricsConfig>,
+    pub acc_consistency_grpc_client: Arc<AccoountConsistencyGrpcClientMetricsConfig>,
+
+    pub found_missing_bubblegums: Gauge,
+    pub found_missing_accounts: Gauge,
+}
+
+impl Default for Peer2PeerConsistencyMetricsConfig {
```

Comment:
It seems to me such default has no usage since `new()` can be freely called instead. Seems like only one should survive imo.

From my perspective, new() with no parameters = default (if the function doesn't provoke side effects)